source/vibe/db/postgresql/package.d

```diff
@@ -89,7 +88,6 @@ class PostgresClient
     {
         logDebugV("get connection from the pool");
         scope conn = pool.lockConnection();
-        scope(exit) destroy(conn);
```

Comment:
Sorry to get back to you so late. Yes, it's guaranteed by the language. The destructor cannot be called after the scope is gone, because the memory is gone. And the destructor is guaranteed to be called. Just to ease any concern, I set this up: https://run.dlang.io/is/aNyM93

> someone faced with it before (as I remember)

If you can point me at the report, I can look into it.